MFVNeutralino/interface/AnalysisConstants.h

```diff
@@ -85,11 +114,11 @@ namespace mfv {
     int_lumi_nice_2017p8 = "  100 fb^{-1} (13 TeV)";
```

Comment:
The luminosity numbers don't seem to match the golden JSON legacy values in https://twiki.cern.ch/twiki/bin/viewauth/CMS/LumiRecommendationsRun2#2018, we should be sure to update all of these.